methods/upload.js

```diff
@@ -0,0 +1,10 @@
+const s3put = require('s3put');
+module.exports = function(file, callback) {
+  const s3 = this.settings.app.s3;
+  const options = {
+    public: true,
+    bucket: s3.bucket,
+    profile: s3.profile
```

Comment:
s3put directly uses the env variables for these if no profile option is provided.